eq-author-api/utils/defaultMetadata.js

```diff
@@ -23,13 +27,13 @@ const defaultValues = [
     key: "ru_name",
     alias: "Ru Name",
     type: "Text",
-    value: "ESSENTIAL ENTERPRISE LTD.",
+    value: "APPLE COMPUTER LIMITED",
```

Comment:
Are we sure we want these to be the default values? If there is no confirmation to change them can we leave them as before.

Reply:
So leave the `defaultValues` as they were but change the `defaultBusinessValues` to use existing `defaultValues`?